server/Dockerfile.build

```diff
@@ -2,6 +2,7 @@ FROM centos
 
 RUN yum -y install docker
 RUN yum -y install git
+RUN yum install -y PyYAML libyaml
```

Comment:
Is it possible to install packages in one `RUN` instruction? Just to avoid additional layers.